### PR TITLE
feat: Add active external Visio connector to profile property settings list - EXO-69942_68649

### PR DESCRIPTION
**Pull request by IlhemEssaadi**
This commit adds the external Visio connectors active for users to the list of profile property settings.

### CODE REVIEW

File: services/src/main/java/org/exoplatform/webconferencing/externalvisio/service/impl/ExternalVisioConnectorServiceImpl.java [hunk 1]
```diff
@@ -31,18 +34,24 @@
...
+  public ExternalVisioConnectorServiceImpl(ExternalVisioConnectorDAO externalVisioConnectorDAO,
+                                           ProfilePropertyService profilePropertyService) {
     this.externalVisioConnectorDAO = externalVisioConnectorDAO;
+    this.profilePropertyService = profilePropertyService;
 
   }
 
   @Override
-  public ExternalVisioConnector createExternalVisioConnector(ExternalVisioConnectorEntity externalVisioConnectorEntity) {
+  public ExternalVisioConnector createExternalVisioConnector(ExternalVisioConnectorEntity externalVisioConnectorEntity) throws ObjectAlreadyExistsException {
```
Comment (rdenarie) [inline services/src/main/java/org/exoplatform/webconferencing/externalvisio/service/impl/ExternalVisioConnectorServiceImpl.java]: why throwing the error here ?
if the setting property exists, so it is ok, catch the error, log a warn, and continue.
No need to propagate the error

File: services/src/main/java/org/exoplatform/webconferencing/externalvisio/service/impl/ExternalVisioConnectorServiceImpl.java [hunk 2]
```diff
@@ -76,7 +85,8 @@ private void processUpdatedExternalVisioConnectors(List<ExternalVisioConnector>
                       });
   }
 
-  public ExternalVisioConnector updateExternalVisioConnector(ExternalVisioConnectorEntity externalVisioConnectorEntity) throws ObjectNotFoundException {
+  public ExternalVisioConnector updateExternalVisioConnector(ExternalVisioConnectorEntity externalVisioConnectorEntity) throws ObjectNotFoundException,
```
Comment (rdenarie) [inline services/src/main/java/org/exoplatform/webconferencing/externalvisio/service/impl/ExternalVisioConnectorServiceImpl.java]: same here, no need to propagate the errror

**Review by rdenarie — CHANGES_REQUESTED** [review]
I think the part about deactivation is missing.
Or do you make it in another PR

**Comment by IlhemEssaadi** [comment]
> I think the part about deactivation is missing. Or do you make it in another PR

Already treated in the updateExternalVisioConnector method.

**Review by rdenarie — APPROVED** [review]

**Pull request merged**